docs/app/pages/components/components-sections/input-controls/number-picker/number-picker.component.html

```diff
@@ -48,8 +48,8 @@
         Can be used to show a red outline around the input to indicate an invalid value. By default the
         error state will appear if the user enters a number below the minimum value or above the maximum value.
     </tr>
-    <tr uxd-api-property name="precision" type="number" defaultValue="6 (maximum 6 digits) or JavaScript MAX_SAFE_INTEGER length">
-        Define the precision of floating point values.
+    <tr uxd-api-property name="precision" type="number" defaultValue="null">
```

Comment:
Normally we just omit the default value if it is null. In this case it won't be `null` if its set the the length of a maximum number by default, however - in this case its not a simple data type, so perhaps just leave the defaultValue off?